sample-tests/src/test/resources/boyka-config.json

```diff
@@ -0,0 +1,56 @@
+{
+  "ui": {
+    "timeout": {
+      "implicit_wait": 10,
+      "explicit_wait": 30,
+      "page_load_timeout": 30,
+      "script_timeout": 10
+    },
+    "screenshot": {
+      "enabled": true,
+      "path": "./screenshots",
+      "extension": "jpeg",
+      "prefix": "SCR"
+    },
+    "web": {
+      "test_local_chrome": {
+        "browser": "CHROME",
+        "headless": "true",
```

Comment:
Use headless value as boolean instead of string.

Reply:
Also check the config documentation and test the settings accordingly.

Reply:
updated